docs/usage/configuration-options.md

```diff
@@ -2828,6 +2828,8 @@ It is recommended that you avoid using "negative" globs, like `**/!(package.json
 
 ### matchDepNames
 
+This field behaves the same as `matchPackageNames` except it matches against `depName` instead of `packageName`.
+
 ### matchDepPatterns
```

Comment:
should we add the deprecated message to those config options? so it's visible for new users to use the preferred matcher?

Reply:
Just to docs you mean? I don't want millions of deprecating log warnings when we can migrate them easily soon

Reply:
yes, just docs only deprecation

Reply:
Done